packages/template-retail-react-app/app/pages/checkout/index.test.js

```diff
@@ -21,7 +21,9 @@ import {
 } from '@salesforce/retail-react-app/app/mocks/mock-data'
 import mockConfig from '@salesforce/retail-react-app/config/mocks/default'
 
-jest.setTimeout(30000)
+// This is a flaky test file!
+jest.retryTimes(5)
+jest.setTimeout(40_000)
```

Comment:
@wjhsf what's up with `_` here?

```suggestion
jest.setTimeout(40000)
```

Reply:
Breaks in certain browsers, doesn't feel worth the tradeoff

https://suprabhasupi.medium.com/numeric-separators-in-javascript-21873dbc0ea#:~:text=Numeric%20Separators%20in%20javascript%2C%20enables,%2C%20octal%2C%20and%20hex%20numbers.

<img width="877" alt="image" src="https://github.com/SalesforceCommerceCloud/pwa-kit/assets/47546998/edff7c91-524e-4cc9-a7ca-20353a54ac6b">

Reply:
Ah right, this is in a test, not browser in this case. that's fine